### DISCUSSION TITLE
About zigbee network design

### DISCUSSION THREAD

**Discussion by vuokko** [Koenkk/zigbee2mqtt#17661] [General]
Hello,

I have quite many (50+) Ikea lights and I wish to move from old Tradfri gateway to something better.

One thing which I like in the old gateway is that it can be turned off and lights still work from remote controls. So the gateway is extra thing. Is this possible network?

And is it possible to somehow adopt the existing network without spending a day re-pairing the devices?

I know that during pairing the network key is sent as cleartext and if whatever gateway I choose (probably SMLight slzb-06) so in theory the gateway could join as router and listen and learn the network and when I make the change to coordinator I would shutdown the old gateway and new gateway would appear and everything would work like before.

But there is probably more security involved.

Or is this something that zigbee2mqtt can't help and I should ask from SMLight?

**Comment by lux73** [comment]
> And is it possible to somehow adopt the existing network without spending a day re-pairing the devices?

not possible - you have to re-pairing all Devices if you want to switch to z2m AFAIK
